docs/user-guide/server.rst

```diff
@@ -1,8 +1,9 @@
+======
```

Comment:
This is valid, but it goes against the heading doc convertions. Look at https://github.com/pulp/pulp/blob/master/docs/user-guide/troubleshooting.rst for example.

I think getting rid of all the ---- ==== adjustments in this PR would be good.